MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/labs/CalendarIntegrationActivity.java

```diff
@@ -206,20 +208,25 @@ public void onResponse(Call<GeocodingResponse> call,
           if (results.size() > 0) {
             // Get the first Feature from the successful geocoding response
             CarmenFeature feature = results.get(0);
-            if (feature != null && style.isFullyLoaded()) {
```

Comment:
Shall we remove the `style` argument from the `makeMapboxGeocodingRequest` method signature since we are not using it?

Reply:
Removed